frontend/templates/product/sections/CompatibilityNotesSection/index.tsx

```diff
@@ -11,12 +11,15 @@ const MAX_DEFAULT_VISIBLE_DEVICES = 10;
 export function splitCompatibilityNotes({
    compatibilityNotes,
 }: CompatibilityNotesSectionProps) {
-   let devices = compatibilityNotes?.trim().split(/\r?\n/) ?? [];
-   devices = devices.map((device) => device.trim()).filter(Boolean);
-   const visibleDevices = devices
-      .slice(0, MAX_DEFAULT_VISIBLE_DEVICES)
-      .join(', ');
-   const hiddenDevices = devices.slice(MAX_DEFAULT_VISIBLE_DEVICES).join(', ');
+   let devices;
+   if (Array.isArray(compatibilityNotes)) {
+      devices = compatibilityNotes;
+   } else {
+      devices = compatibilityNotes?.trim().split(/\r?\n/) ?? [];
+      devices = devices.map((device) => device.trim()).filter(Boolean);
```

Comment:
Huh... I don't think I've ever done `.filter(Boolean)` but I think I like it more than `.filter((x) => x))`. Nice.